prebuilt/common/su.d/50viper.sh

```diff
@@ -0,0 +1,2 @@
+#! /system/bin/sh
+/system/xbin/supolicy --live "allow mediaserver mediaserver_tmpfs:file { read write execute };"
```

Comment:
I think I picked the commit on mm6.0 branch. Will add it on 6.0.1 and remove this.